pkg/agent/agent.go

```diff
@@ -641,3 +645,26 @@ func getRoundInfo(bridgeClient ovsconfig.OVSBridgeClient) types.RoundInfo {
 
 	return roundInfo
 }
+
+func (i *Initializer) getNodeMTU(localIntf *net.Interface) (int, error) {
+	if i.networkConfig.DefaultMTU != 0 {
+		return i.networkConfig.DefaultMTU, nil
+	}
+	mtu := localIntf.MTU
+	if mtu <= 0 {
```

Comment:
could you add a comment to explain when this case could happen (or if it is not supposed to happen ever, but you feel like we should keep it, maybe add a comment to say that this is just defensive-programming)

Reply:
MTU is assigned via DHCP option. If for whatever reason, its not set, this sanity check handles the case.